src/network/networkd-dhcp4.c

```diff
@@ -1876,6 +1882,36 @@ int config_parse_dhcp_fallback_lease_lifetime(const char *unit,
         return 0;
 }
 
+int config_parse_dhcp_label(
+                const char *unit,
+                const char *filename,
+                unsigned line,
+                const char *section,
+                unsigned section_line,
+                const char *lvalue,
+                int ltype,
+                const char *rvalue,
+                void *data,
+                void *userdata) {
+
+        char **s = data;
+
+        assert(filename);
+        assert(lvalue);
+        assert(rvalue);
+        assert(data);
+
+        if (!address_label_valid(rvalue)) {
+                log_syntax(unit, LOG_WARNING, filename, line, 0,
+                           "Address label is too long or invalid, ignoring assignment: %s", rvalue);
+                return 0;
+        }
+        log_syntax(unit, LOG_WARNING, filename, line, 0,
+                   "TOTO config_parse_dhcp_label %s", rvalue);
+
+        return free_and_strdup_warn(s, empty_to_null(rvalue));
```

Comment:
`address_label_valid()` refuses an empty string, so it is not necessary to wrap `rvalue` with `empty_to_null()`.

Reply:
I reimplement `config_parse_label` based on [config_parse_hostname](https://github.com/systemd/systemd/blob/main/src/network/networkd-network.c#L850)
I don't use `empty_to_null()` anymore.